src/components/side_panel/pivot/pivot_layout_configurator/pivot_layout_configurator.ts

```diff
@@ -261,12 +261,13 @@ export class PivotLayoutConfigurator extends Component<Props, SpreadsheetChildEn
   addCalculatedMeasure() {
     const { measures }: { measures: PivotCoreMeasure[] } = this.props.definition;
     const measureName = this.env.model.getters.generateNewCalculatedMeasureName(measures);
+    const aggregator = "sum";
     this.props.onDimensionsUpdated({
       measures: measures.concat([
         {
-          id: this.getMeasureId(measureName),
+          id: this.getMeasureId(measureName, aggregator),
```

Comment:
Not sure about the fix. I'm thinking that in the future we might try to guess the aggregator with a `split(":")` on the measure id ... but there is no migration or anything, so old spreadsheet will just break without us noticing

Reply:
Old spreadsheets already have ids without the aggregate...
It's already granted that the ids could be anything and we can't infer anything from them